spec/rubocop/cop/rspec/file_fixture_spec.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::FileFixture, :config do
+  it 'registers an offense when using ' \
+     '`"\#{Rails.root}/spec/fixtures/files/some_file.csv"`' do
+    expect_offense(<<~RUBY)
+      "\#{Rails.root}/spec/fixtures/files/some_file.csv"
```

Comment:
I think it's possible to write heredoc as `<<~'RUBY'` to avoid escaping the hash.